jre/java/java/util/List.java

```diff
@@ -164,4 +166,8 @@ default Spliterator<E> spliterator() {
   }
 
   @JsNonNull List<E> subList(int fromIndex, int toIndex);
+
+  static <E> List<E> copyOf(Collection<? extends E> coll) {
+    return coll.stream().collect(Collectors.toUnmodifiableList());
+  }
```

Comment:
This is too costly esp for code size (bringing in stream implementation). Let's rely on toArray + Collections.internalListOf

Reply:
@gkdn something like that:
```
 static <E> List<E> copyOf(Collection<? extends E> coll) {
    return Collections.internalListOf((E[]) ArrayHelper.unsafeClone(coll.toArray(), 0, coll.size()));
  }
```
right?

Reply:
toArray returns a new copy so we don't need to clone; so something like:
```
static <E> List<E> copyOf(Collection<? extends E> coll) {
  return Collections.internalListOf((E[]) coll.toArray());
}
```

(and similar pattern for the others)